modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -101,6 +101,7 @@ public void elasticsearchOssImage() throws IOException {
                      DockerImageName
                          .parse("docker.elastic.co/elasticsearch/elasticsearch-oss")
                          .withTag(ELASTICSEARCH_VERSION)
+                         .asCompatibleSubstituteFor("docker.elastic.co/elasticsearch/elasticsearch")
```

Comment:
this is an interesting edge case. Ideally, both images should we treated as compatible automatically. WDYT?

Reply:
Good suggestion - will have a try.

Reply:
I've modified `assertCompatibleWith` to accept a varargs array of `DockerImageName`s, and I think this works a lot better.